AI/CMakeLists.txt

```diff
@@ -27,6 +32,9 @@ if(NOT fuzzylite_FOUND)
     set(FL_BUILD_BINARY OFF CACHE BOOL "")
     set(FL_BUILD_SHARED OFF CACHE BOOL "")
 	set(FL_BUILD_TESTS OFF CACHE BOOL "")
+	if("x${CMAKE_CXX_COMPILER_FRONTEND_VARIANT}" STREQUAL "xGNU" OR ${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU")
+		add_compile_options(-Wno-error=deprecated-declarations)
```

Comment:
I would rather fix warnings, but OK, can be fixed later in separate PR

Reply:
We will fix warnings in FuzzyLite? I disabled it because it is FuzzyLite header warnings, not VCMI one.

Reply:
If this is about fuzzylite then better to add comment about that.
But you also have one in library makefile.
Anyway, right now goal should be making final build work on Windows, warning can be dealt with later

Reply:
@IvanSavenko It is about FARPROC, and this is explained in commit message.

Reply:
done